src/main/java/com/sap/oss/phosphor/fosstars/tool/github/SingleSecurityRatingCalculator.java

```diff
@@ -5,144 +5,83 @@
 import com.sap.oss.phosphor.fosstars.data.UserCallback;
 import com.sap.oss.phosphor.fosstars.data.ValueCache;
-import com.sap.oss.phosphor.fosstars.data.github.CodeqlDataProvider;
-import com.sap.oss.phosphor.fosstars.data.github.FuzzedInOssFuzz;
-import com.sap.oss.phosphor.fosstars.data.github.GitHubDataFetcher;
-import com.sap.oss.phosphor.fosstars.data.github.HasBugBountyProgram;
-import com.sap.oss.phosphor.fosstars.data.github.HasCompanySupport;
-import com.sap.oss.phosphor.fosstars.data.github.HasSecurityPolicy;
-import com.sap.oss.phosphor.fosstars.data.github.HasSecurityTeam;
-import com.sap.oss.phosphor.fosstars.data.github.InfoAboutVulnerabilities;
-import com.sap.oss.phosphor.fosstars.data.github.IsApache;
-import com.sap.oss.phosphor.fosstars.data.github.IsEclipse;
-import com.sap.oss.phosphor.fosstars.data.github.LgtmDataProvider;
-import com.sap.oss.phosphor.fosstars.data.github.NumberOfCommits;
-import com.sap.oss.phosphor.fosstars.data.github.NumberOfContributors;
-import com.sap.oss.phosphor.fosstars.data.github.NumberOfStars;
-import com.sap.oss.phosphor.fosstars.data.github.NumberOfWatchers;
-import com.sap.oss.phosphor.fosstars.data.github.OwaspSecurityLibraries;
-import com.sap.oss.phosphor.fosstars.data.github.PackageManagement;
-import com.sap.oss.phosphor.fosstars.data.github.ProgrammingLanguages;
-import com.sap.oss.phosphor.fosstars.data.github.SignsJarArtifacts;
-import com.sap.oss.phosphor.fosstars.data.github.UsesDependabot;
-import com.sap.oss.phosphor.fosstars.data.github.UsesFindSecBugs;
-import com.sap.oss.phosphor.fosstars.data.github.UsesGithubForDevelopment;
-import com.sap.oss.phosphor.fosstars.data.github.UsesNoHttpTool;
-import com.sap.oss.phosphor.fosstars.data.github.UsesOwaspDependencyCheck;
-import com.sap.oss.phosphor.fosstars.data.github.UsesSanitizers;
-import com.sap.oss.phosphor.fosstars.data.github.UsesSignedCommits;
-import com.sap.oss.phosphor.fosstars.data.interactive.AskAboutSecurityTeam;
-import com.sap.oss.phosphor.fosstars.data.interactive.AskAboutUnpatchedVulnerabilities;
-import com.sap.oss.phosphor.fosstars.model.RatingRepository;
+import com.sap.oss.phosphor.fosstars.model.Rating;
 import com.sap.oss.phosphor.fosstars.model.ValueSet;
-import com.sap.oss.phosphor.fosstars.model.rating.oss.OssSecurityRating;
 import com.sap.oss.phosphor.fosstars.model.subject.oss.GitHubProject;
 import com.sap.oss.phosphor.fosstars.model.value.ValueHashSet;
-import com.sap.oss.phosphor.fosstars.nvd.NVD;
-import java.io.IOException;
-import java.util.Arrays;
 import java.util.Comparator;
 import java.util.List;
 import java.util.Objects;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
 /**
- * The class calculates a security rating for a single open-source project.
+ * The class calculates a rating for a project.
  */
-class SingleSecurityRatingCalculator {
+public class SingleSecurityRatingCalculator implements RatingCalculator {
```

Comment:
Maybe I'm wrong, but should the class be renamed in e.g. `SingleRatingCalculator`?

```suggestion
public class SingleRatingCalculator implements RatingCalculator {
```